boards/cc2538dk/include/periph_conf.h

```diff
@@ -59,33 +59,29 @@ static const timer_conf_t timer_config[] = {
 /** @} */
 
 /**
- * @name UART configuration
+ * @name    UART configuration
  * @{
  */
-#define UART_NUMOF          1
+static const uart_conf_t uart_config[] = {
+    {    /* Virtual COM Port */
```

Comment:
What does `/* Virtual COM Port */` mean here? Configuring an UART is neither virtual, nor is it a (Windows) COM port in this context :-)

Reply:
Well I'm with @haukepetersen, this comment should be removed or updated (e.g USB port or something more hardware related).

Reply:
Sorry, have removed that now!